base/docs/helpdb/Base.jl

```diff
@@ -2972,6 +3097,12 @@ midpoints
     .+(x, y)
 
 Element-wise addition operator.
+
+```jldoctest
+julia> [1 2 3] .+ [1 2 4]
```

Comment:
do we have examples somewhere of the broadcasting behavior? that's more interesting but also more complicated to explain